test/errhandling/psahabu/catch-parens.chpl

```diff
@@ -0,0 +1,19 @@
+use ThrowError;
+
+try {
+  throwAnError();
+} catch () {
```

Comment:
Yeah this is weird, I'll disallow it. Splitting up empty and non-empty is also appealing because `newTemp(dtVoid)` doesn't survive until `errorHandling` due to the void type feature, so I can just create a separate case for that.